keps/sig-node/2712-pod-priority-based-graceful-node-shutdown/kep.yaml

```diff
@@ -15,16 +15,17 @@ prr-approvers:
   - "@deads2k"
 
 # The target maturity stage in the current dev cycle for this KEP.
-stage: alpha
+stage: beta
 
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.23"
+latest-milestone: "v1.24"
 
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
   alpha: "v1.23"
+  beta: "v1.24"
```

Comment:
This KEP appears to have gaps in the monitoring and troubleshooting sections.  Please update with how to know this feature is working, how to know this feature is breaking at scale, and how debug this feature.